src/main/java/net/masterthought/cucumber/util/Status.java

```diff
@@ -13,7 +13,8 @@
     SKIPPED("#88AAFF"),
     PENDING("#FBB907"),
     UNDEFINED("#FBB957"),
-    MISSING("#FBB9A7");
+    MISSING("#FBB9A7"),
+    HIDDEN("");
```

Comment:
I want to get rid of velocity as well - something modem - maybe handlebars

Sent from my iPhone

> On 13 May 2015, at 20:37, Bogdan Livadariu <notifications@github.com> wrote:
> 
> In src/main/java/net/masterthought/cucumber/util/Status.java:
> 
> > @@ -13,7 +13,8 @@
> >      SKIPPED("#88AAFF"),
> >      PENDING("#FBB907"),
> >      UNDEFINED("#FBB957"),
> > -    MISSING("#FBB9A7");
> > +    MISSING("#FBB9A7"),
> > +    HIDDEN("");
> This will require a major refactor :) The view should be handled by the velocity template and the logic by the java, so we would strip the java methods that wraps html code. I think I can come up with something for 0.0.26 release.
> …
> —
> Reply to this email directly or view it on GitHub.
>